client/src/utils/track.ts

```diff
@@ -7,6 +7,14 @@ let configured = false;
 if (!configured) {
   Auth.configure(awsexports);
   Analytics.configure(awsexports);
+  // @ts-ignore
+  if (typeof AWSCShortbread !== "undefined") {
+    // @ts-ignore
+    const shortbread = AWSCShortbread({
```

Comment:
Do we need to create a new instance as follows? 
```suggestion
    const shortbread = new AWSCShortbread({
```

Reply:
Strangely no, it's `AWSCShortbread({...})` -- not sure why 😅